### ISSUE TITLE
[diode-send-dir] Missing option 

### ISSUE THREAD

**Issue by julieenn** [CrabeDeFrance/lidi#14]
Hi,

When I tried to use `diode-send-dir` (with the v2.1.0 tag), I found that the options in the documentation didn't match the options offered by the binary. 

`diode-send-dir` in the doc : 
```
Usage: diode-send-dir [OPTIONS] <DIR>

Options:
      --to-tcp <TO_TCP>                IP address and port to connect in TCP to diode-send (ex "127.0.0.1:5001") [default: 127.0.0.1:5001]
      --buffer-size <BUFFER_SIZE>      Size of file buffer [default: 8196]
      --hash                           Compute a hash of file content (default is false)
      --dir <DIR>                      Directory containing files to send
      --maximum-files <MAXIMUM_FILES>  maximum number of files to send per session
      --log-config <LOG_CONFIG>        Path to log configuration file
      --debug...                       Verbosity level. Using it multiple times adds more logs
      --help                           Print help
      --version                        Print version
```

Compiled  `diode-send-dir` :
```
$ ./diode-send-dir --help
Usage: diode-send-dir [OPTIONS] <DIR>

Arguments:
  <DIR>  Directory containing files to send

Options:
      --to-tcp <TO_TCP>                IP address and port to connect in TCP to diode-send (ex "127.0.0.1:5001") [default: 127.0.0.1:5001]
      --buffer-size <BUFFER_SIZE>      Size of file buffer [default: 8196]
      --hash                           Compute a hash of file content (default is false)
      --maximum-files <MAXIMUM_FILES>  maximum number of files to send per session
      --log-config <LOG_CONFIG>        Path to log configuration file
      --log-level <LOG_LEVEL>          Verbosity level: info, debug, warning, error ... [default: info]
  -h, --help                           Print help
  -V, --version                        Print version
```

Note that the binary will not work with the `--dir` option (we could have imagined that the `--help` would just not be up to date, but that's not the case here). 
```
$ ./diode-send-dir --dir /appli/lidi/input
error: unexpected argument '--dir' found

  tip: to pass '--dir' as a value, use '-- --dir'

Usage: diode-send-dir [OPTIONS] <DIR>

For more information, try '--help'.
```

Yours sincerely.
Julien

**Comment by CrabeDeFrance** [comment]
Hello, thanks for your comment, I will update the documentation.

**Comment by CrabeDeFrance** [comment]
Should be fixed in version 2.2.0

**Comment by julieenn** [comment]
I can confirm that this has been resolved in version 2.2.0.

Thanks a lot. 

Julien
